### ISSUE TITLE
Reproduce you result on TE1. The MAE is much worse than yours on DUTS-TE

### ISSUE THREAD

**Issue by YiChuanChang** [Karel911/TRACER#2]
Hi,
I tried your code to reproduce your result. 
I used `python main.py train --arch 1 --img_size 320` to train the model. I did not change any setting in your code. I used four GPU to train this model. But the MAE I got on DUTS-TE is around 0.088 (it is 0.033 in paper). In picture, it shows the loss and the MAE of train and val set when training. It early stopped at 82th epoch. It kind of converged. 

I would like to know did you do any data augmentation off-line (I train on DUTS-TR and the edge gt you provided. I simply change the folder directory to meet your folder structure mentioned in readme)? Should I change any hyper parameter to achieve the same result?
![image](https://user-images.githubusercontent.com/13060645/147037215-177cf8a6-e549-4359-9e7c-d468e8c120b8.png)

**Comment by Karel911** [comment]
Hi,
There was some minor issues at the initializing models.
I have checked the mentioned code and updated the `util/effi_utils.py`.
Now the model can converge more faster.

I restored the pre-trained TRACER-Efficient-1 (as I answered #1 ) and obtained the same results listed at the original paper.

![results](https://user-images.githubusercontent.com/30099757/147044853-9fa83292-4dcb-44a4-b531-1a7d857194fe.png)

**Comment by YiChuanChang** [comment]
Thanks for responding me so fast.
Yeah, there is no problem to restored the pre-trained TRACER-Efficient-1 and get the same MAE.
I just want to train the model by myself and see how it works.
I will try your new code. I will let you know if I get anything new.

**Comment by YiChuanChang** [comment]
[In line 17 in TRACER](https://github.com/Karel911/TRACER/blob/main/model/TRACER.py#L17) you used `advprop=True` but later on you did not pass this into the [load_pre_trained_weights](https://github.com/Karel911/TRACER/blob/main/model/EfficientNet.py#L316) Is this in purpose?

**Comment by Karel911** [comment]
Thank you for raising careful issues.
The arguments `weights_path=None` and `advprop=True` were missed when we revised the code for public...
The default settings are `weights_path=None` and `advprop=True`.
We have updated the [load_pretrained_weights](https://github.com/Karel911/TRACER/blob/main/model/EfficientNet.py#L316).

**Comment by YiChuanChang** [comment]
Thanks for your help. The result is the same with the pre-trained model you provided now. It is pretty amazing the pre-trained backbone model helped the training a lot. (Blur: w pertained backbone, Orange: w/o pertained backbone)
![image](https://user-images.githubusercontent.com/13060645/147178935-c7f724d0-ccb1-4308-91c4-dadf95bbf272.png)
